tests/testthat/test-ResultsDataModel.R

```diff
@@ -49,7 +49,17 @@ test_that("Create schema", {
 
 test_that("Results upload", {
   skip_if(skipResultsDm | skipCdmTests, 'results data model test server not set')
-
+  if (dbms == "sqlite") {
+    # Checks to see if adding extra OMOP vocab, unexpectedly breaks things
+    connection <- DatabaseConnector::connect(connectionDetails)
+    with_dbc_connection(connection, {
+      DatabaseConnector::renderTranslateExecuteSql(connection, "
+        INSERT INTO main.vocabulary
```

Comment:
@msuchard I added this test because, looking at the branch, it seems that the VA's CDM is violating the primary key constraint of the vocabulary table?

I'm hesitant to support this in general - I'm not sure why one CDM would have multiple OMOP standards - but I don't think this will have any negative impacts for CohortDiagnostics

Reply:
I'm not going to support or refute the merit of what the VA VINCI team is doing with their `cdm_source` table.   Their `vocabulary` tables are actually views; this may be part of the issue.

Nonetheless, their `cdm_source` table does contain multiple rows, upon which `CohortDiagnostics` currently fails.  So, if we want `CohortDiagnostics` be less brittle and keep friends happy ....

Reply:
if its only cdm_source - why dont we just do a max for this one table? 

we dont need the info from this table for "diagnostics"

Reply:
that way we dont have to second guess or make any changes to the other areas of the code